loops-and-arrays.js

```diff
@@ -20,6 +25,11 @@ const names = ["Henk", "Piet", "Fred", "Joop"];
 // ==========================================
 
 const numbers = [2, 4, 5, 29, 38];
+console.log(numbers);
+
+for(let i = 0; i < numbers.length; i++)
```

Comment:
ik mis de opening en closing tag van je forloop { } 

hij logt hem wel goed, dat is dan wel weer grappig. hierbij moet de console.log ook weer buiten de forloop worden geplaatst